src/Files.App/Dialogs/SettingsDialog.xaml

```diff
@@ -6,133 +6,135 @@
-						CornerRadius="0"
-						Tag="4">
-						<NavigationViewItem.Icon>
-							<FontIcon Glyph="&#xF1AD;" />
-						</NavigationViewItem.Icon>
-					</NavigationViewItem>
-					<NavigationViewItem
-						AccessKey="B"
-						AutomationProperties.AutomationId="SettingsItemAbout"
-						Content="{helpers:ResourceString Name=About}"
-						Tag="5">
-						<NavigationViewItem.Icon>
-							<FontIcon FontSize="16" Glyph="&#xE946;" />
-						</NavigationViewItem.Icon>
-					</NavigationViewItem>
-				</NavigationView.MenuItems>
+			<!--  Menu Items  -->
+			<NavigationView.MenuItems>
+				<NavigationViewItem
+					AccessKey="A"
+					AutomationProperties.AutomationId="SettingsItemAppearance"
+					Content="{helpers:ResourceString Name=Appearance}"
+					IsSelected="True"
+					Tag="0">
+					<NavigationViewItem.Icon>
+						<FontIcon HorizontalAlignment="Left" Glyph="&#xE790;" />
+					</NavigationViewItem.Icon>
+				</NavigationViewItem>
+				<NavigationViewItem
+					AccessKey="P"
+					AutomationProperties.AutomationId="SettingsItemPreferences"
+					Content="{helpers:ResourceString Name=SettingsNavPreferences/Content}"
+					Tag="1">
+					<NavigationViewItem.Icon>
+						<FontIcon Glyph="&#xE9E9;" />
+					</NavigationViewItem.Icon>
+				</NavigationViewItem>
+				<NavigationViewItem
+					AccessKey="F"
+					AutomationProperties.AutomationId="SettingsItemFolders"
+					Content="{helpers:ResourceString Name=Folders}"
+					Tag="2">
+					<NavigationViewItem.Icon>
+						<FontIcon Glyph="&#xE8B7;" />
+					</NavigationViewItem.Icon>
+				</NavigationViewItem>
+				<NavigationViewItem
+					AccessKey="T"
+					AutomationProperties.AutomationId="SettingsItemTags"
+					Content="{helpers:ResourceString Name=FileTags}"
+					Tag="3">
+					<NavigationViewItem.Icon>
+						<FontIcon Glyph="&#xE1CB;" />
+					</NavigationViewItem.Icon>
+				</NavigationViewItem>
+				<NavigationViewItem
+					AccessKey="E"
+					AutomationProperties.AutomationId="SettingsItemAdvanced"
+					Content="{helpers:ResourceString Name=Advanced}"
+					Tag="4">
+					<NavigationViewItem.Icon>
+						<FontIcon Glyph="&#xF1AD;" />
+					</NavigationViewItem.Icon>
+				</NavigationViewItem>
+				<NavigationViewItem
+					AccessKey="B"
+					AutomationProperties.AutomationId="SettingsItemAbout"
+					Content="{helpers:ResourceString Name=About}"
+					Tag="5">
+					<NavigationViewItem.Icon>
+						<FontIcon FontSize="16" Glyph="&#xE946;" />
+					</NavigationViewItem.Icon>
+				</NavigationViewItem>
+			</NavigationView.MenuItems>
 
-				<ScrollViewer
-					x:Name="SettingsScrollViewer"
-					VerticalAlignment="Stretch"
-					VerticalScrollBarVisibility="Visible"
-					VerticalScrollMode="Enabled">
-					<Frame x:Name="SettingsContentFrame" Padding="12" />
-				</ScrollViewer>
-			</NavigationView>
-		</Grid>
-	</Border>
+			<!--  Content Frame  -->
+			<ScrollViewer x:Name="SettingsContentScrollViewer">
+				<Frame
+					x:Name="SettingsContentFrame"
+					Padding="16"
```

Comment:
Why was this changed?

Reply:
Actually, should be 24px and 16px for small window

![image](https://user-images.githubusercontent.com/62196528/224179223-29798005-608d-4347-992f-2443ab2be1c4.png)

Reply:
This is a small window if you measure the width of the page.

Reply:
@mdtauk Yeah I agree. At this time, we should limit the scope of purpose of the PR.